src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inversRobot = {};
+  const keys = Object.keys(robot);
+  const values = Object.values(robot);
+
+  for (let i = 0; i < keys.length; i++) {
+    if (inversRobot.hasOwnProperty(values[i])) {
+      return null;
+    } else {
+      inversRobot[values[i]] = keys[i];
+    }
```

Comment:
`else` redundant here

```suggestion
    }
    
      inversRobot[values[i]] = keys[i];
   
```